### PR TITLE
rhcos-packages: Make bootupd no arch dependent

### PR DESCRIPTION
**Pull request by ravanelli**
We now have bootupd support across all architectures.

### CODE REVIEW

File: common.yaml [hunk 1]
```diff
@@ -242,6 +242,8 @@ packages:
  - kernel-modules-extra
  # Audit
  - audit
+ # Bootloader updater
+ - bootupd
```
Comment (jlebon) [inline common.yaml:246]: We already include the same `bootupd.yaml` manifest in this file. We just need to move it out from the `arch-include` and into the `include` section.

BTW, to confirm the no-op s390x support will also correctly no-op on the `bootupctl backend generate-update-metadata` step?

**Comment by jlebon** [comment]
/hold

We can't do this yet because bootupd is too ancient on 9.2 to ship on ppc64le and s390x, which is still the default variant built and shipped. So for now this needs to go in 9.4-specific manifests.

**Comment by dustymabe** [comment]
newer version of this in https://github.com/openshift/os/pull/1439

I think 4.16 is 9.4 now so what was holding up this PR shouldn't be an issue any longer?

**Comment by travier** [comment]
/close
Closing in favor of https://github.com/openshift/os/pull/1439

**Pull request closed without merging**